### ISSUE TITLE
Specific accessibility acceptance criteria for stories

### ISSUE THREAD

**Issue by petermarks-gsa** [GSA-TTS/FAC#234]
**Background**: We opened this issue as it's unclear what should be accessibility tested manually and what's done automatically. 

**Conversation with @heymatthenry about current automated CI checks:** Ax Core, HTML code sniffer, lighthouse (including accessibility audit) - This catches pretty much anything that should be wrong with the markup (correct use of ARIA, contrast ratios) 

**Specific accessibility acceptance criteria:**
* Listen to the experience with a screen reader - is the information presented in order
* Navigate the page with tabs - are you able to get the job done?

**Comment by petermarks-gsa** [comment]
@heymatthenry ask I discussed this (see description for notes) and I have updated the [story issue template](https://github.com/GSA-TTS/FAC/issues/templates/edit) to specify what manual checks are needed. Tagging @MelissaBraxton and @austinhernandez to review this.

**Comment by MelissaBraxton** [comment]
This sounds good as a starting point. Are we clear on who does this testing and when and how it happens in the story lifecycle?

**Comment by petermarks-gsa** [comment]
@MelissaBraxton Per our accepted [agile process](https://github.com/GSA-TTS/FAC/blob/main/docs/agile-process.md), I the PM will be the one doing this testing on each story before moving to done. I saw Austin closed this. Do you think there needs to be more refinement or are we good here?

**Comment by austinhernandez** [comment]
One thing I am not sure is caught is to test content scaling since we are primarily relying on browser controls

**Comment by petermarks-gsa** [comment]
@austinhernandez What specific content scaling test cases are you thinking will be helpful? Like "Test with 120% browser scaling" or "Test with 80% browser scaling"? Also, is it fair to assume that we are essentially not officially supporting mobile for new audit submissions?

**Comment by austinhernandez** [comment]
From https://webaim.org:
- For WCAG 2.0 - Zoom to 200%
- For WCAG 2.1 - Adjust viewport to 1280 pixels wide and zoom to 400%. Avoid horizontal scrolling unless required for the content (e.g., a data table or map).
- Text-only zoom (Recommended) - In Firefox, choose View > Zoom > Zoom Text Only. Zoom to 130-150%

I can't imagine a case for mobile for upload, but even on the current IMS it works on mobile. Mobile considerations should be there and are part of the USWDS so they usually are, what do you think @MelissaBraxton ?

**Comment by austinhernandez** [comment]
I also added some acceptance criteria examples on #143 that I copied here that I would love other designer feedback on:

**Acceptance criteria examples for design ticket**
- [ ]  All text has a contrast ratio of 4.5:1 with the background
- [ ]  Interaction are documented if they deviate from USWDS components
- [ ]  The needs of someone navigating the site with the use of a screenreader or keyboard has been considered and documented 
- [ ] There is an option for multilingual support
- [ ] There is supporting documentation for alt text, labels for links (anchor + target), error text in forms, and what assistive technologies should announce (if applicable)


**For a stories about research:**
- [ ]  User research participant recruiting includes work with users with disabilities
- [ ] Writing research plans and protocols for users with disabilities

**Comment by MelissaBraxton** [comment]
LGTM!

**Comment by petermarks-gsa** [comment]
Conversation at standup today:

@heymatthenry - For [this issue](https://github.com/GSA-TTS/FAC/issues/127), it's an issue with a USWDS component

@MelissaBraxton - Post in accessibility channel and reach out to Robert Jolly

**Comment by petermarks-gsa** [comment]
Recap of conversation with Robert Jolly at design sync, who is experienced in accessibility matters:

[USWDS only supports WCAG 2.0 AA](https://designsystem.digital.gov/documentation/developers/#accessibility), not WCAG 2.1. Recommendation is to shoot for WCAG 2.0 AA, but raise WCAG 2.1 issues with USWDS. I've gone ahead and changed this line to:

- [ ] Text scaling - Adjust viewport to 1280 pixels wide and zoom to 200%, ensure everything renders as expected. Document 400% zoom issues with USWDS if appropriate. 

I've updated the process [docs](https://github.com/GSA-TTS/FAC/pull/250) and @MelissaBraxton approved the PR, so going to consider this issue resolved.
